website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
About multiple authors, I'm all with @rvalyi. Each person, from the same company or other, has to be included in the header of the file when his/her contribution is significant. About the word "Copyright", it was issued some time ago, but there's a lot of code inside OCA that uses the abbreviation. Personally, I don't care about that word, because in open source is very meaningless (in contrary to atribution), but if it's a blocking one, we can change. To use the abbreviation is to save chars for PEP8.